src/Libraries/Nop.Core/Domain/StoreInformationSettings.cs

```diff
@@ -52,5 +52,10 @@ public class StoreInformationSettings : ISettings
         /// Gets or sets a value of YouTube channel URL of the site
         /// </summary>
         public string YoutubeLink { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value of Instagram account URL of the site
+        /// </summary>
+        public string InstagramLink { get; set; }

```

Comment:
This setting must be filled during installation, like other social media links. Let's use the link to our official account by default (https://www.instagram.com/nopcommerce_official). And don't forget about migration for this setting.